src/ExpressReceiver.spec.ts

```diff
@@ -66,29 +155,49 @@ describe('ExpressReceiver', () => {
     // ----------------------------
     // runWithValidRequest
 
-    async function runWithValidRequest(req: Request, errorResult: any) {
+    async function runWithValidRequest(req: Request, state: any) {
       // Arrange
       const resp = {} as Response;
-      const next = (error: any) => { errorResult = error; };
+      const next = (error: any) => { state.error = error; };
 
       // Act
       const verifier = verifySignatureAndParseBody(noopLogger, signingSecret);
       await verifier(req, resp, next);
-      return errorResult;
     }
 
     it('should verify requests', async () => {
-      let errorResult: any;
-      runWithValidRequest(buildExpressRequest(), errorResult);
+      const state: any = {};
```

Comment:
The tests I added before failed to detect failures while parsing the body. This pull request also fixes it.